PerfettoCds/Pipeline/Tables/PerfettoGenericEventTable.cs

```diff
@@ -0,0 +1,218 @@
+﻿using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Diagnostics.CodeAnalysis;
+using PerfettoCds.Pipeline.DataOutput;
+using Microsoft.Performance.SDK;
+using PerfettoCds.Pipeline.DataCookers;
+
+namespace PerfettoCds.Pipeline.Tables
+{
+    [Table]
+    public class PerfettoGenericEventTable
+    {
+        // Set some sort of max to prevent ridiculous field counts
+        public const int AbsoluteMaxFields = 20;
+
+        public static TableDescriptor TableDescriptor => new TableDescriptor(
+            Guid.Parse("{37cedaaa-5679-4366-b627-9b638aaefeb3}"),
```

Comment:
Didn't check but make sure all these table and column GUIDs below are unique and re-generated for Perfetto